std_msgs/Header header
geometry_msgs/Point32[] lane_left_points
geometry_msgs/Point32[] lane_right_points
